rotkehlchen/accounting/constants.py

```diff
@@ -126,6 +126,19 @@
     HistoryEventType.BURN: {
         HistoryEventSubType.NFT: {DEFAULT: EventCategory.BURN_NFT},
     },
+    HistoryEventType.MULTI: {
```

Comment:
We had a convo about the naming here: https://discord.com/channels/657906918408585217/1364225036621316096/1364693201688203316

Probably best to add more subtypes to keep the hierarchy of main events clear.